Tests/Unit/LoRaWan/DevAddrTests.cs

```diff
@@ -3,13 +3,40 @@
 
 namespace LoRaWan.Tests.Unit
 {
+    using System;
     using LoRaWan;
+    using LoRaWan.Tests.Common;
     using Xunit;
 
     public class DevAddrTests
     {
         private readonly DevAddr subject = new(0xeb6f7bde);
 
+        [Theory]
+        [InlineData(uint.MaxValue)]
+        [InlineData(uint.MinValue)]
+        [InlineData(0xeb6f7bde)]
+        public void Init(uint value)
+        {
+            // arrange
+            var maxValue = new DevAddr(value);
+
+            // act
+            var result = new DevAddr(maxValue.NetworkId, maxValue.NetworkAddress);
+
+            // assert
+            Assert.Equal(maxValue, result);
```

Comment:
This seems to be a test for equality rather than initialization.

Reply:
Addressed this with 01e2056508a2ea255e01e447bf07fce5aa20ecc4